lib/service/service.go

```diff
@@ -3988,9 +3988,17 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 	transportpb.RegisterTransportServiceServer(sshGRPCServer, transportService)
 
 	process.RegisterCriticalFunc("proxy.ssh", func() error {
+		// If a ssh proxy address is specified provide the address and specifics
+		sshListenerAddr := listeners.ssh.Addr().String()
+		sshListenerAddrDetails := sshListenerAddr
+		if cfg.Proxy.SSHAddr.Addr != "" {
+			sshListenerAddr = cfg.Proxy.SSHAddr.Addr
+			// will include address and type (tcp)
+			sshListenerAddrDetails = fmt.Sprintf("%v", cfg.Proxy.SSHAddr)
```

Comment:
```suggestion
			sshListenerAddrDetails = cfg.Proxy.SSHAddr.String()
```

Reply:
Sure, I've simplified and removed the consolef output

Reply:
Looks much better. What do you think about removing consolef completely?

Reply:
Removed all from service startup logging.  I added one log for auth service since it only had consolef

Reply:
Can we remove the function itself too?

Reply:
The function, tests and all usage have been removed